Add Finances page with grant administration details (#20)
tomeshnet/toronto-community-network-docs: src/operations/finances.md

# Finances

The purpose of this document is to describe the financial status and processes of the Toronto Community Network.

## Introduction
The Toronto Community Network is a community project that current does not have its own independent legal entity or bank accounts, and relies on its fiscal hosting partners to provide fiscal administration capacity.

> Aside from out-of-pocket spendings from some of our members, our current financing comes from one single grant from the [Internet Society](https://www.internetsociety.org) administering through [Free Geek Toronto](https://www.freegeektoronto.org).

Reviewer: Could be considered a false positive. It indicates instances of  spending.

However could also be written as an uncountable noun.

```suggestion
Aside from out-of-pocket spending from some of our members, our current financing comes from one single grant from the [Internet Society](https://www.internetsociety.org) administering through [Free Geek Toronto](https://www.freegeektoronto.org).
```

Reviewer 2: "fiscal hosting partners" the prevalent term in the nonprofit sector is "trustee"

## Internet Society Grant
TBA (some information about the grant)

### Administration

> Free Geek Toronto currently holds the full amount of the grant, and the balance of X (after 10% of administrative fees) is to be distributed for the project according to the budget set out in the grant proposal.

Reviewer: Possibly include the two scheduled deposits as well @TimTor

Reviewer 2: ```suggestion
Instead of "according to the budget set out in the grant proposal." suggest "according to the Memorandum of Understanding with Free Geek Toronto." since this doc is about the relationsship with FGT rather than ISOC
```

2. Free Geek Toronto will approve allocations according to whether an expense approved by the Administrators contains required documentations for their accounting purposes, and they reserve the right to reject allocations that may put them in a position of legal liability due to violation of grant terms.

In the course of administering the grant, Administrators may draft request for proposals (RFPs) or delegate budgets to working groups to allocate as they deem necessary.
For example, the Administrators may allocate a budget for training and request for member applications, or delegate a small budget for the _Communications and Community Engagement_ working group to spend on an event as they see fit.

Hardware purchase makes up the bulk of the Internet Society grant budget.
The _Network Planning, Design and Operations_ working group will be responsible for making hardware procurement proposals using the following template, submitted as a pull request to [operations/hardware-budgets](https://github.com/tomeshnet/toronto-community-network/new/master/operations/hardware-budgets/):

> ```

Reviewer 3: ```suggestion
```markdown
```
Marking this as markdown will allow syntax highlighting for the rendered book.

Toronto Mesh estimated a contribution of 1630 member hours of volunteer labour to the project.

Toronto Mesh and Free Geek Toronto are expected to deliver 3 reports by the following dates:
- 1st report: September 30, 2020
- 2nd report: November 15, 2020
- Final report: January 31, 2020 (summarizing all the work done until December 31, 2020)

Four core members of Toronto Mesh, who will serve as Administrators for this grant (explained below), have signed a Memorandum of Understanding (MoU) with Free Geek Toronto to formalize the fiscal hosting relationship.
The full Internet Society Partnership Agreement and MoU are [archived on our trusted drive](https://cloud.tomesh.net/apps/files/?dir=/tomeshnet-trusted/ISOC), accessible only to core members due to confidentiality requirements from our partners.

### Administration
The first of two installments in the amount of CAD 7,000 is paid to Free Geek Toronto in September of 2020, and the balance of CAD 6,300 (after 10% of administrative fees) is to be distributed for the project according to the budget set out in the grant proposal.
We have two-levels of approvals for fund allocations:

> 1. Four members of Toronto Mesh involved in the grant application are selected as Administrators, and will approve allocations according to whether an expense fits with the spirit of the grant objectives and its proposed budget.

Author: I would like to propose for @darkdrgn2k @TimTor @Shrinks99 and myself as the Administrators of this grant from Toronto Mesh. Is this okay with you guys?

The 4 of us will sign the MoU and take on the job of reviewing expenses moving forward. I expect these to be straightforward as the budget is pretty well defined.

@ASoTNetworks @makeworld-the-better-one you two are the people who were also involved in the original TCN proposal, I want to know if this arrangement is okay with you guys too.

I explicitly do not want @ASoTNetworks to be an Administrator because @YurkoWasHere @ASoTNetworks and myself are all part of @hyphacoop and @Shrinks99 can better represent Communications WG. @makeworld-the-better-one can't be Administrator right?

Reviewer 4: This makes sense to me, I agree with the points about Hypha and I'm happy to sign it!

Reviewer 3: This arrangement works for me, thanks!

Reviewer 2: Sounds good to me
